src/openrct2/scripting/ScTile.hpp

```diff
@@ -112,6 +112,17 @@ namespace OpenRCT2::Scripting
             Invalidate();
         }
 
+        uint8_t baseZ_get() const
+        {
+            return _element->GetBaseZ();
+        }
+        void baseZ_set(uint8_t value)
```

Comment:
Should be an uin16_t at least. Valid range is `0 - (254 * COORDS_Z_STEP)`. (Because 255 is used to mark the tile element as discarded.

Reply:
Same for clearanceZ.
Changed it to uint16_t.